drawing-board/cooking-loose/terraform.tfvars

```diff
@@ -2,14 +2,30 @@ resource_group_name = "cl-dev-rg"
 
 location = "EastUS"
 
-admin_username = "cladmin"
-
-admin_password = "G0ld5t4r!"
-
 vnet_name = "cl-dev-vnet"
 
 vnet_address_space = ["192.168.0.0/16"]
 
+# Global VM vars
```

Comment:
This all looks much tidier and I can see the utility of it very well.

One thing that I'm curious about: do we actually not need the quote marks on the subvar names (on the left)?

Reply:
Not sure actually. Let's test it!